# Analysis Notebook

In [18]:
# Importing all the libraries necessary for this Project

import pandas as pd
import numpy as np
import math
import matplotlib.pyplot as plt
import seaborn as sns

In [19]:
# importing dataset

Sales = pd.read_csv("Cleaned_Sales.csv")

In [20]:
Sales.head(20)

Unnamed: 0  Rank                                          Name Platform  \
0            0     1                                    Wii Sports      Wii   
1            1     2                             Super Mario Bros.      NES   
2            2     3                                Mario Kart Wii      Wii   
3            3     4                             Wii Sports Resort      Wii   
4            4     5                      Pokemon Red/Pokemon Blue       GB   
5            5     6                                        Tetris       GB   
6            6     7                         New Super Mario Bros.       DS   
7            7     8                                      Wii Play      Wii   
8            8     9                     New Super Mario Bros. Wii      Wii   
9            9    10                                     Duck Hunt      NES   
10          10    11                                    Nintendogs       DS   
11          11    12                                 Mario Kart DS       DS   
12          12    13                   Pokemon Gold/Pokemon Silver       GB   
13          13    14                                       Wii Fit      Wii   
14          14    15                                  Wii Fit Plus      Wii   
15          15    16                            Kinect Adventures!     X360   
16          16    17                            Grand Theft Auto V      PS3   
17          17    18                 Grand Theft Auto: San Andreas      PS2   
18          18    19                             Super Mario World     SNES   
19          19    20  Brain Age: Train Your Brain in Minutes a Day       DS   

      Year         Genre               Publisher  NA_Sales  EU_Sales  \
0   2006.0        Sports                Nintendo     41.49     29.02   
1   1985.0      Platform                Nintendo     29.08      3.58   
2   2008.0        Racing                Nintendo     15.85     12.88   
3   2009.0        Sports                Nintendo     15.75     11.01   
4   1996.0  Role-Playing                Nintendo     11.27      8.89   
5   1989.0        Puzzle                Nintendo     23.20      2.26   
6   2006.0      Platform                Nintendo     11.38      9.23   
7   2006.0          Misc                Nintendo     14.03      9.20   
8   2009.0      Platform                Nintendo     14.59      7.06   
9   1984.0       Shooter                Nintendo     26.93      0.63   
10  2005.0    Simulation                Nintendo      9.07     11.00   
11  2005.0        Racing                Nintendo      9.81      7.57   
12  1999.0  Role-Playing                Nintendo      9.00      6.18   
13  2007.0        Sports                Nintendo      8.94      8.03   
14  2009.0        Sports                Nintendo      9.09      8.59   
15  2010.0          Misc  Microsoft Game Studios     14.97      4.94   
16  2013.0        Action    Take-Two Interactive      7.01      9.27   
17  2004.0        Action    Take-Two Interactive      9.43      0.40   
18  1990.0      Platform                Nintendo     12.78      3.75   
19  2005.0          Misc                Nintendo      4.75      9.26   

    JP_Sales  Other_Sales  Global_Sales  
0       3.77         8.46         82.74  
1       6.81         0.77         40.24  
2       3.79         3.31         35.82  
3       3.28         2.96         33.00  
4      10.22         1.00         31.37  
5       4.22         0.58         30.26  
6       6.50         2.90         30.01  
7       2.93         2.85         29.02  
8       4.70         2.26         28.62  
9       0.28         0.47         28.31  
10      1.93         2.75         24.76  
11      4.13         1.92         23.42  
12      7.20         0.71         23.10  
13      3.60         2.15         22.72  
14      2.53         1.79         22.00  
15      0.24         1.67         21.82  
16      0.97         4.14         21.40  
17      0.41        10.57         20.81  
18      3.54         0.55        

## Top 3 Genre of Games that were sold the most

In [21]:
Sales["Genre"].value_counts().head(3)

Genre
Action    3251
Sports    2304
Misc      1686
Name: count, dtype: int64

According to the results The Most Sold 'Genre' is 'Action' with a massive lead with a difference of 949 with the 2nd place 'Sports' and on the 3d position is Miscelanious

### In what year the Highest number of Action Gmes were sold?

In [22]:
Sales[Sales['Genre']=='Action'].groupby(by=['Year'])['Year'].value_counts().sort_values(ascending=False).head(5)

Year
2009.0    272
2012.0    265
2015.0    255
2011.0    239
2010.0    226
Name: count, dtype: int64

Action Games were sold most in year 2009 and since then the Trend of action Genre has gone up as compared to past.

### What was the Name of the Game highest sold all over the world in 'Action Genre'?

In [23]:
Sales[Sales['Genre']=='Action'].sort_values(by=['Global_Sales'], ascending=False).head(5)

Unnamed: 0  Rank                           Name Platform    Year   Genre  \
16          16    17             Grand Theft Auto V      PS3  2013.0  Action   
17          17    18  Grand Theft Auto: San Andreas      PS2  2004.0  Action   
23          23    24             Grand Theft Auto V     X360  2013.0  Action   
24          24    25    Grand Theft Auto: Vice City      PS2  2002.0  Action   
38          38    39           Grand Theft Auto III      PS2  2001.0  Action   

               Publisher  NA_Sales  EU_Sales  JP_Sales  Other_Sales  \
16  Take-Two Interactive      7.01      9.27      0.97         4.14   
17  Take-Two Interactive      9.43      0.40      0.41        10.57   
23  Take-Two Interactive      9.63      5.31      0.06         1.38   
24  Take-Two Interactive      8.41      5.49      0.47         1.78   
38  Take-Two Interactive      6.99      4.51      0.30         1.30   

    Global_Sales  
16         21.40  
17         20.81  
23         16.38  
24         16.15  
38         13.10

Video Games Sold globally HIghest in Action Category is "Grand Theft Auto V"

## Which Year had highest sales in Video game market

In [24]:
Sales["Year"].value_counts().head(12)

Year
2009.0    1431
2008.0    1428
2010.0    1257
2007.0    1201
2011.0    1136
2006.0    1008
2005.0     936
2002.0     829
2003.0     775
2004.0     744
2012.0     655
2015.0     614
Name: count, dtype: int64

We can observe the era of Gaming reached its peak during 2009.

The Sale of Games showed an exponential growth around 2005-2006 and soon fell rapidly.

### What kind of Games were sold in 2009 the highest?

In [25]:
Sales[Sales['Year']==2009].groupby(by=['Genre'])['Genre'].value_counts().sort_values(ascending=False).head(5)

Genre
Action        272
Misc          207
Sports        184
Adventure     141
Simulation    123
Name: count, dtype: int64

Highest Genre of Games Sold in 2009 was 'Action'

## What kind of Platforms were used to buy Games the most

In [26]:
Sales["Platform"].value_counts().head(5)

Platform
DS      2131
PS2     2127
PS3     1304
Wii     1290
X360    1234
Name: count, dtype: int64

We can observe that Nintendo DS was the PLatform with the most selling games till date.
Playstation-2 is also not behind and is nearly equal to Nintendo DS and they can be considered nearly tied on the first place.

### In  what Year did the 'DS' Games were sold the most?

In [27]:
Sales[Sales['Platform']=='DS'].groupby(by=['Year'])['Year'].value_counts().sort_values(ascending=False).head(5)

Year
2008.0    492
2009.0    405
2007.0    377
2010.0    326
2006.0    202
Name: count, dtype: int64

In 2008 Highest Number Of 'DS' (492) copies of games were sold

### What kind of Games were being played on 'DS' the most?

In [28]:
Sales[Sales['Platform']=='DS'].groupby(by=['Genre'])['Genre'].value_counts().sort_values(ascending=False).head(5)

Genre
Misc          389
Action        335
Simulation    280
Adventure     238
Puzzle        236
Name: count, dtype: int64

As we Can see the HIghest Type of Games played on DS Platform were of Miscelaneous Types and following with Action